offsite_payments.gemspec

```diff
@@ -22,9 +22,19 @@ Gem::Specification.new do |s|
   s.files = Dir['CHANGELOG', 'README.md', 'MIT-LICENSE', 'lib/**/*']
   s.require_path = 'lib'
 
+  unless defined?(Money)
```

Comment:
We should mention the money gem needs to be added before this gem in the Gemfile.